sources/packages/backend/apps/queue-consumers/src/processors/schedulers/student-application-notifications/student-application-notifications.scheduler.ts

```diff
@@ -39,7 +44,28 @@ export class StudentApplicationNotificationsScheduler extends BaseScheduler<void
         `Processing student application notifications job. Job id: ${job.id} and Job name: ${job.name}.`,
       );
 
-      // TODO: Get applications that have a disability status mismatch  and check PDPPD status
+      const eligibleApplications =
+        await this.applicationService.getEligibleApplicationsForNotification();
+
+      for (const application of eligibleApplications) {
+        const notification: StudentPDPPDNotification = {
+          userId: application.student.user.id,
+          givenNames: application.student.user.firstName,
+          lastName: application.student.user.lastName,
+          email: application.student.user.email,
+          applicationNumber: application.applicationNumber,
+        };
+
+        await this.notificationActionsService.saveStudentApplicationPDPPDNotification(
+          notification,
+          application.currentAssessment.id,
+        );
+      }
+
+      this.logger.log(JSON.stringify(eligibleApplications));
+      processSummary.info(
+        `Eligible applications: ${JSON.stringify(eligibleApplications)}`,
+      );
```

Comment:
Please refrain from dumping entire objects in the logs unless requested/necessary (it does not seem to be the case).

Reply:
Unless really required we should avoid adding user/student personal data to logs.
The logs can be as below.
```ts
if (assessments.length) {
  processSummary.info(
    `PD/PPD mismatch assessments that generated notifications: ${assessments.join(
      ",",
    )}`,
  );
} else {
  processSummary.info(
    `No assessments found to generate PD/PPD mismatch notifications.`,
  );
}
```